test/integration.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-env mocha */
```

Comment:
Tried it, but doesn't seem to work with the `standard` CLI. On the other hand it is (at least for now) just a single file, so I think it's fine for now.